ios-base/Onboarding/ViewModels/FirstViewModel.swift

```diff
@@ -43,15 +43,16 @@ class FirstViewModel {
       return
     }
     //This fails with 404 since this endpoint is not implemented in the API base
-    UserService.sharedInstance.loginWithFacebook(
-      token: token.tokenString,
-      success: { [weak self] in
-        self?.state = .network(state: .idle)
-        AppNavigator.shared.navigate(to: HomeRoutes.home, with: .changeRoot)
-      },
-      failure: { [weak self] error in
-        self?.state = .network(state: .error(error.localizedDescription))
-    })
+    //TODO: remove when implementing login with apple id
```

Comment:
I would say, remove it now. No need to let Facebook as dependency there if in the near future Apple Sign-in will be there

Reply:
I can do that in a follow-up PR, this one is pretty big already

Reply:
Cool. No need to do it here. But for sure no need to wait until Apple Sign-in is ready